src/mWater_collate/sampling_spreadsheet_creator.R

```diff
@@ -0,0 +1,21 @@
+sampling_spreadsheet_creator <- function(date_oi = "2023-10-16" ){
+
+  #pull in site meta data
```

Comment:
quibble: I like to add lines between steps/chunks of code to improve readability.